src/kixi/nhs/board_report.clj

```diff
@@ -7,7 +7,58 @@
             [kixi.nhs.data.transform                       :as transform]
             [kixi.nhs.patient-experience.deprivation       :as deprivation]
             [kixi.nhs.patient-experience.ethnicity         :as ethnicity]
-            [kixi.nhs.patient-experience.gender-comparison :as gender]))
+            [kixi.nhs.patient-experience.gender-comparison :as gender]
+            [clj-time.format                               :as tf]
+            [clj-time.core                                 :as t]
+            [clj-time.coerce                               :as tc]))
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;; Idicator 65: Incidence of MRSA                                                       ;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+;; TODO this formatter will be changed when bug in CKAN is fixed and data re-uploaded
+(def custom-formatter (tf/formatter "yyyy-MM-dd'T'HH:mm:ss"))
+
+(defn latest-month
+  "Finds the latest month in data.
+  It looks for a date using provided key.
+  Returns a datetime."
+  [k data]
+  (->> data
+       (map #(tf/parse custom-formatter (k %)))
+       t/latest))
+
+(defn enrich
+  "Use latest month timestamp to enrich the map
+  with year and period_of_coverage which are PKs in
+  datastore.
+  Stringifies the sum value."
+  [timestamp m]
+  (-> m
+      (assoc :year (str (t/year timestamp))
+             :period_of_coverage (tf/unparse custom-formatter timestamp))
+      (update-in [:sum] str)))
+
+(defn incidence
```

Comment:
I just missed the comment block above. This is on the borderline of needing to be in a separate namespace, but I think is OK for now.